theano/gpuarray/extra_ops.py

```diff
@@ -450,12 +450,13 @@ def c_support_code_struct(self, node, nodename):
         """ % locals()
         return super(GpuCumsum, self).c_support_code_struct(node, nodename) + code
 
-
+@register_opt('fast_compile')
 @op_lifter([CumsumOp])
-def use_gpu_cumsumop(node, ctx_name):
-    if node.inputs[0].dtype == 'float32':
-        axis = node.op.axis
-        x = node.inputs[0]
+@register_opt2([CumsumOp], 'fast_compile')
```

Comment:
Don't register like you suggest in your comment. I think the code is good. How can I reproduce the problem to investigate?